io.github.maddecoder.Classic-RBDOOM-3-BFG.yml

```diff
@@ -0,0 +1,160 @@
+app-id: io.github.maddecoder.Classic-RBDOOM-3-BFG
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "23.08"
+command: DOOMBFA.sh
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.mono6
+build-options:
+    append-path: /usr/lib/sdk/mono6/bin
+    append-ld-library-path: /usr/lib/sdk/mono6/lib
+    append-pkg-config-path: /usr/lib/sdk/mono6/lib/pkgconfig
+
+finish-args:
+- --device=all
+- --socket=wayland
+- --socket=fallback-x11
+- --share=ipc
+- --share=network
+- --socket=pulseaudio
+- --filesystem=~/.var/app/com.valvesoftware.Steam:ro
+- --filesystem=~/.var/app/net.lutris.Lutris:ro
+- --filesystem=~/.var/app/com.heroicgameslauncher.hgl:ro
+- --filesystem=~/.doombfa:create
```

Comment:
Can this be `persist=.doombfa`? This creates the folder inside the sandbox, but bind mounts it to relative to real home.

Reply:
I want to let the user to be able to access that folder either for save backup, bugfixing (through the log file or by tinkering the settings file) and for the installation of an addon.
Can the user do all those while is on persist? Also the engine expects the save folder to be on $HOME/.doombfa (not just the shell script) will that create any issues with that?

Reply:
> Can the user do all those while is on persist?

Yea it will be in `~/.var/app/io.github.maddecoder.classic-rbdoom-3-bfg/.doombfa`

I don't think it'll have issues, but you should check locally

Reply:
I tried yesterday, and the shell script didn't seem to like it. Should I change the path? If so to what?

Reply:
So I used XDG_DATA on the shell script and it passed. But then the engine was unable to write to the save folder. So it might be better to keep it that way (plus it's easier for the user to find it)

Reply:
After some changes to the main project, it now works with persist